main.go

```diff
@@ -84,10 +89,10 @@ func main() {
 	watchNamespace, _ := os.LookupEnv(watchNamespaceEnvVar)
 
 	controllerOptions := ctrl.Options{
-		Namespace:              watchNamespace,
-		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
```

Comment:
Port seems to be deprecated, wasn't in the metricsserver options or the controller options, testing it out with a basic deployment on minikube, didn't really break anything from what I've seen, I'm hoping to be corrected by someone though

Reply:
Ah found it, will push a commit